Gem/Code/Source/Components/Multiplayer/EnergyCannonComponent.cpp

```diff
@@ -12,6 +12,7 @@
 #include <AzCore/Settings/SettingsRegistry.h>
 #include <Source/Components/Multiplayer/EnergyBallComponent.h>
 #include <Source/Components/Multiplayer/EnergyCannonComponent.h>
+#include <Components/PerfTest/NetworkPrefabSpawnerComponent.h>
```

Comment:
Out of curiousity, are components in "PerfTest" meant to have general-purpose use like this? If so, should it maybe get moved out of PerfTest?

Reply:
I had that originally because I copied the spawning logic from the gem components. The spawn ticket system had a lot of drawbacks however, and I was concerned about general correctness and safety, so I reverted to using the standard network entity manager paths. Doing so simplified things quite nicely, all the weird unresolved issues around entity ownership went away, and there's no more dependency on this perftest component.